gatsby/content/faq/encryption.mdx

```diff
@@ -15,12 +15,12 @@ End-to-End Encryption is currently in late beta. Rooms can have encryption enabl
 
 End-to-end encryption is currently available in:
 
-- [Riot Web](https://github.com/vector-im/riot-web/)
+- [Element Web](https://github.com/vector-im/element-web/)
   - and [matrix-js-sdk](https://github.com/matrix-org/matrix-js-sdk)
   - and [matrix-react-sdk](https://github.com/matrix-org/matrix-react-sdk)
-- [Riot Android](https://github.com/vector-im/riot-android/)
+- [Element Android](https://github.com/vector-im/element-android/)
   - and <a href="https://github.com/matrix-org/matrix-android-sdk">matrix-android-sdk</a>
```

Comment:
```suggestion
  - and <a href="https://github.com/matrix-org/matrix-android-sdk2">matrix-android-sdk</a>
```

Reply:
Thanks. Seems like the name is changed too, looking at this: https://github.com/matrix-org/matrix-android-sdk2#how-to-integrate-the-sdk-in-your-application